src/main/java/uk/gov/hmcts/reform/bsp/common/service/PostcodeValidator.java

```diff
@@ -10,15 +10,15 @@
         List<String> validationMessages = new ArrayList<>();
         if (fieldsMap.containsKey(postcodeKey)) {
             String postcodeValue = fieldsMap.get(postcodeKey);
-            if (isPostcodeLengthInvalid(postcodeValue)) {
+            if (!isPostcodeLengthValid(postcodeValue)) {
```

Comment:
isPositiveCondition is better idea IMO (and in opinion of Clean Code author ;) )